DEVELOP.md

```diff
@@ -73,6 +73,7 @@ gulp                    # Compile the JS and start the automatic
 To regenerate the `bundled.js` file without using the live reloader, run:
 
 ```
+gulp peg
```

Comment:
> My general preference is to just remember one command that always results in a correct state

That's what I'm suggesting with sticking `peg` into the `default` task; we're on the same page here :) 

@armish Sorry that such a small change is getting outsized discussion, but I agree that sticking the `peg` task into the `default` task list is the best way to handle this. Then we don't need to mention it in the README/DEVELOP.md at all.